### DISCUSSION TITLE
How can a local plugin hide it's models from the Collection Manager

### DISCUSSION THREAD

**Discussion by Nazaire** [strapi/strapi#7809] [Legacy Topics]
Hi guys i've only just started to get going with Strapi. 

I'm trying to create a local plugin with a model where I don't want the Admin UI to be able to manage items in the collection. I'll set the collection items up myself, internally. It's very similar to how strapi-plugin-user-permissions works, where there is an interface for editing 'User' but the other models 'Role' and 'Permission' are hidden. I've been looking into [the plugin](https://github.com/strapi/strapi/tree/master/packages/strapi-plugin-users-permissions) to try and see how it does this, but I can't find anything. **How does the user-permissions plugin specify which of it's models should & should not be shown in the collection types menu?**

Cheers,
Dylan

**Comment by Nazaire** [comment]
After doing some investigation it looks like it's hard coded into the [content-manager plugin](https://github.com/strapi/strapi/tree/cd9296b37bca030aa1c56e4782a28f1ce79a24f8/packages/strapi-plugin-content-manager) 
https://github.com/strapi/strapi/blob/cd9296b37bca030aa1c56e4782a28f1ce79a24f8/packages/strapi-plugin-content-manager/services/ContentTypes.js#L25

**Reply by derrickmehaffy** [reply]
I believe this is the case, but I will ping @soupette and @HichamELBSI to see if we have something that isn't documented.

**Comment by soupette** [comment]
@derrickmehaffy there's no option in the back-end yet to hide a local plugin from the manager.
